src/jank/clojure/core.jank

```diff
@@ -927,6 +927,25 @@
          ret (reduce f init coll)]
      (f ret))))
 
+;; TODO: private
+(defn preserving-reduced
```

Comment:
We can use `^:private` here, since jank supports meta hints now. It just won't be marked as private; I don't think I'll implement private vars in jank, since Clojure does a halfway job of them and ClojureScript doesn't bother.